libraries/standard/json/cbmc/proofs/JSON_Validate/Makefile

```diff
@@ -0,0 +1,13 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HARNESS_FILE=JSON_Validate_harness
+
+CBMC_MAX_BUFSIZE=14
+
+UNWINDSET += JSON_Validate.0:$(CBMC_MAX_BUFSIZE)
+
+include ../Makefile-json.common
+
+$(PROOFDIR)/json.c: $(SRCDIR)/libraries/standard/json/src/json.c
+	sed -E '1s/^/#include "glue.h" /; s/^static //; s/(bool_|JSONStatus_t|void) skip(AnyLiteral|Collection|Number|Space|String)\b/&_/' $< > $@
```

Comment:
It changes the storage class of the functions from static to extern.  It also appends an underscore to a subset of functions so that their replacements will be used instead (the originals are proven in another proof).

Why?  The JSON validator is too big to prove all at once, so I did it in cumulative steps.